README.md

```diff
@@ -641,6 +641,88 @@ Set up a dual auth delete policy and also a policy to rotate the key every 2 mon
             AllowedIPPort responseObj = response.getResult();
 ```
 
+### Key alias
+
+###### Create key alias
+```
+            // Construct an instance of the CreateKeyAliasOptions model
+            CreateKeyAliasOptions createKeyAliasOptionsModel = new CreateKeyAliasOptions.Builder()
+                    .id(keyId)
+                    .alias(keyAlias)
+                    .bluemixInstance(bluemixInstance)
+                    .build();
+
+            // Invoke operation with valid options model, response status code should be 201
+            Response<KeyAlias> response = testService.createKeyAlias(createKeyAliasOptionsModel).execute();
+
+```
+
+###### Get key by using key alias
+```
+            // Construct an instance of the GetKeyOptions model
+            GetKeyOptions getKeyOptions = new GetKeyOptions.Builder().id(keyAlias)
+                    .bluemixInstance(bluemixInstance)
+                    .build();
+
+            // Invoke operation with valid options model
+            Response<GetKey> response = testService.getKey(getKeyOptions).execute();
+            GetKey responseObj = response.getResult();
+```
+
+###### Delete key alias
+```
+            // Construct an instance of the DeleteKeyAliasOptions model
+            DeleteKeyAliasOptions deleteKeyAliasOptionsModel = new DeleteKeyAliasOptions.Builder()
+                    .id(keyId)
+                    .alias(keyAlias)
+                    .bluemixInstance(bluemixInstance)
+                    .build();
+
+            // Invoke operation with valid options model, response status code should be 204
+            Response<Void> response = testService.deleteKeyAlias(deleteKeyAliasOptionsModel).execute();
+
+```
+
+### Key ring
+
```

Comment:
Probably want to make sure we have an example of how to create a key inside a key ring